src/nvim/api/command.c

```diff
@@ -557,10 +557,12 @@ String nvim_cmd(uint64_t channel_id, Dict(cmd) *cmd, Dict(cmd_opts) *opts, Error
     }
 
     if (HAS_KEY(mods.tab)) {
-      if (mods.tab.type != kObjectTypeInteger || mods.tab.data.integer < 0) {
-        VALIDATION_ERROR("'mods.tab' must be a non-negative Integer");
+      if (mods.tab.type != kObjectTypeInteger) {
+        VALIDATION_ERROR("'mods.tab' must be a Integer");
```

Comment:
```suggestion
        VALIDATION_ERROR("'mods.tab' must be an Integer");
```

Reply:
I used the same as the one below, so I'll change both.